ci/jenkins/pipelines/skuba-upgrade-from-4.2.Jenkinsfile

```diff
@@ -0,0 +1,45 @@
+pipeline {
+
+    agent none
+    
+    stages {
+        stage('upgrade') {
+            agent { label "${WORKER_LABEL}" }
+            
+            environment {
+                REG_CODE = credentials('scc-regcode-caasp')
+            }
+            steps {
+                echo 'upgrading'
```

Comment:
I assume we should provision and deploy a cluster here and then run the upgrade steps. Am I correct?

Reply:
Actually there are additional steps: build the skuba, provision the cluster, deploy the cluster (v4.2). Those steps are part of a previous stage